web/templates/settings/admin_human_form.hbs

```diff
@@ -29,7 +29,7 @@
     <div class="input-group">
         {{#if is_active}}
         <span {{#if owner_is_only_user_in_organization}}class="deactivate_user_button_tooltip"{{/if}}>
-            <button class="button rounded button-danger deactivate_user_button" {{#if owner_is_only_user_in_organization}}disabled="disabled"{{/if}}>
+            <button class="action-button action-button-quiet-danger action-button-padding" {{#if owner_is_only_user_in_organization}}disabled="disabled"{{/if}}>
```

Comment:
Please consider using the `web/templates/components/action-button.hbs` component for the action buttons.